### PR TITLE
WQP-1339 add pipeline.yml and Jenkinsfiles

### PR DESCRIPTION
**Pull request by skaymen**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Add pipeline.yml and two Jenkinsfiles (build and deploy).

Description
-----------
https://internal.cida.usgs.gov/jira/browse/WQP-1339

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: Jenkinsfile.build [hunk 1]
```diff
@@ -0,0 +1,78 @@
+@Library('dockerCI') _
```
Comment (ayan-usgs) [inline Jenkinsfile.build]: You'll probably want to tie to a version 0.3.0. The syntax is `dockerCI@0.3.0` or `PipelineLibs@0.3.0`.

File: Jenkinsfile.deploy [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@Library('dockerCI') _
```
Comment (ayan-usgs) [inline Jenkinsfile.deploy]: See comment above.

File: pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+deployJobName: "qw-portal-services-deploy"
+# name of the project
+projectName: "wqp-external"
+# maximum amount of RAM for running Docker containers in the service
+memory: 512
+# name of the application or service
+applicationName: "qw-portal-services"
+# integer representing the order of precedence of the ALB rule.
+# set to a low priority; services are given higher priority.
+rulePriority: 503
```
Comment (ayan-usgs) [inline pipeline.yml]: This number should be unique among the WQP services. 503 will collide with the WQP_UI

File: pipeline.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+# port that is exposed by your container
+containerPort: 5050
```
Comment (ayan-usgs) [inline pipeline.yml]: I think this container actually exposes port `8080`

File: pipeline.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+# string parameter specifying a path that ALB should use to verify that your application is alive
+healthCheck: "/robots.txt"
```
Comment (ayan-usgs) [inline pipeline.yml]: I'm not sure this project even has a robots.txt that can be hit.

File: pipeline.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+# string parameter that tells the ALB how to route requests to the containers (e.g. /my-app/*); this forms a listener rule for the load balancer
+contextPath: "/*"
```
Comment (ayan-usgs) [inline pipeline.yml]: This is going to collide with WQP_UI. Might want to use `/services` or something. Jim or Steinich might have some thoughts on what it should be.

File: pipeline.yml (hunk 5 above)
Comment (mbucknell) [reply pipeline.yml]: I think the health check should be "actuator/health"

**Pull request merged**